core/src/usb.py

```diff
@@ -29,79 +29,92 @@
 ENABLE_IFACE_WEBAUTHN = not utils.BITCOIN_ONLY
 ENABLE_IFACE_VCP = __debug__
 
-# interface used for trezor wire protocol
-id_wire = next(_iface_iter)
-iface_wire = io.WebUSB(
-    iface_num=id_wire,
-    ep_in=0x81 + id_wire,
-    ep_out=0x01 + id_wire,
-    emu_port=UDP_PORT + WIRE_PORT_OFFSET,
-)
-bus.add(iface_wire)
+# All the interfaces are (conditionally) created and added in add_interfaces()
+iface_wire: io.WebUSB | None = None
```

Comment:
AFAIK creating the interfaces (without calling `bus.add()`) should be OK, so please create the interfaces on top level as before, and only add them in the `add_interfaces()` function.

Reply:
Sure. Small problem is that in `add_interfaces()` we will need to check which interfaces are defined.

We could use the same if-statements as when defining the interfaces, even though that would be some duplication. Probably still cleaner than checking something like `if  "iface_debug" in globals():`

Reply:
Done in [3822719](https://github.com/trezor/trezor-firmware/pull/2129/commits/38227199375ace354525361b2879f8b7eee43bc4)

Reply:
yes, reusing the same ifs is OK. that is how the code looked sometime ago actually :)